src/controllers/users.js

```diff
@@ -1,58 +1,77 @@
-import { Router } from 'express';
 import HttpStatus from 'http-status-codes';
 import * as userService from '../services/userService';
-import { findUser, userValidator } from '../validators/userValidator';
-
-const router = Router();
 
 /**
- * GET /api/users
+ * Get all users
```

Comment:
Full stop at the end and a new line after that.

Reply:
🆗